.github/workflows/nox.yml

```
@@ -90,7 +90,7 @@ jobs:
         with:
           path: antsibull-docs-parser
       - name: Setup nox
-        uses: wntrblm/nox@2024.03.02
+        uses: wntrblm/nox@2024.04.15
```

This one likely has to stay:
```suggestion
        uses: wntrblm/nox@2024.03.02
```